src/main/java/com/arcbees/bourseje/shared/CookieNames.java

```diff
@@ -0,0 +1,5 @@
+package com.arcbees.bourseje.shared;
+
+public class CookieNames {
```

Comment:
It's used by IdentificationPresenter and VoteResource. Not the most useful thing ever, but I preferred that to hard-coding in both places.